### PR TITLE
feat: S3 maven migration

### PR DESCRIPTION
**Pull request by inckie**
### Detailed readable description

Migrate from Bintray maven to S3 hosted one.

### Checklist
- [x] I've provided a readable title for the PR
- [x] I've provided a detailed description
- [x] The PR is scoped to a single task/feature
- [ ] I've included relevant tests (if tests are not included please provide the reason why they aren't)

- [x] My PR is not introducing a UI change
- [ ] My PR is introducing UI changes and I provided all screenshots in the PR description

- [x] My PR is a part of a new feature or a feature improvement
    - [ ] I've provided a link in the description to the relevant card in the Zapp cycle feature rollout Trello board.
- [ ] My PR is a bug fix
    - [ ] I provided a link in the description to the relevant Jira ticket  or provided the following in the PR description:
        - steps to reproduce a bug
        - expected result

### CODE REVIEW

File: rake/templates/gradle-wrapper.properties.erb [hunk 1]
```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-all.zip
```
Comment (ranmeirman) [inline rake/templates/gradle-wrapper.properties.erb:5]: is this a breaking change? what will happen to plugins?

**Review by ranmeirman — CHANGES_REQUESTED** [review]

File: rake/templates/gradle-wrapper.properties.erb (hunk 1 above)
Comment (inckie) [reply rake/templates/gradle-wrapper.properties.erb:5]: Lower versions has issues retrieving artifacts from S3, so I bumped it to a minimal one that was mentioned to work OK. Build itself mostly depends on the Android Gradle plugin, that we don't change yet.

Comment (inckie) [reply rake/templates/gradle-wrapper.properties.erb:5]: Issues are on Android Studio only though (it seems to get into loop), CI works in any case.

**Review by ranmeirman — APPROVED** [review]

**Comment by inckie** [comment]
Deleting the branch, SDK is not back on master

**Pull request merged**